src/juicy-popover.html

```diff
@@ -54,6 +54,26 @@
                 case "expandableSelector":
                     this.shadowRoot.querySelector("#expandable content").setAttribute("select", newVal || ".expandable");
                     break;
+
+                case "disabled":
+                    var shouldBeDisabled = (newVal !== null);
+                    if(!this.disabled && shouldBeDisabled) {
+                        this.collapse();
+                    }
+                    this.disabled = shouldBeDisabled;
+                    break;
+
+                case "expanded":
+                    var shouldBeExpanded = (newVal !== null);
+                    if(!this.expanded && shouldBeExpanded) {
+                        this.expand();
+                    }
+                    if(this.expanded && !shouldBeExpanded) {
+                        this.collapse();
+                    }
+
+                    this.expanded = shouldBeExpanded;
+                    break;
```

Comment:
Please add the explanation of `expanded` and `disabled` attributes to `README.md`.